### PR TITLE
Bold the profile name and truncate the pubkey

### PR DESCRIPTION
**Pull request by BeaudanBrown**

### CODE REVIEW

File: ts/components/conversation/ContactName.tsx [hunk 1]
```diff
@@ -10,20 +10,27 @@ interface Props {
...
     const shouldShowProfile = Boolean(profileName && !name);
-    const profileElement = shouldShowProfile ? (
-      <span className={`${prefix}__profile-name`}>
-        <Emojify text={profileName || ''} i18n={i18n} />
-      </span>
-    ) : null;
+    const profileElement = shouldShowProfile ?
+      (boldProfileName ? (
+        <span className={`${prefix}__profile-name`}>
+          <b><Emojify text={profileName || ''} i18n={i18n} /></b>
```
Comment (sachaaaaa) [inline ts/components/conversation/ContactName.tsx]: I don't talk react that well, but maybe there's a way to do
```tsx
if (boldProfile) {
   profileElement = <b> {profileElement} </b>;
}
```
?
Maybe Mikunk can vouch?

Comment (msgmaxim) [reply ts/components/conversation/ContactName.tsx]: Yeah, that should work

File: ts/components/conversation/Message.tsx [hunk 2]
```diff
@@ -307,11 +307,12 @@ export class Message extends React.PureComponent<Props, State> {
     return (
       <div className="module-message__author">
         <ContactName
-          phoneNumber={authorPhoneNumber}
+          phoneNumber={`(...${authorPhoneNumber.substring(authorPhoneNumber.length - 6, authorPhoneNumber.length)})`}
```
Comment (msgmaxim) [inline ts/components/conversation/Message.tsx]: I believe `authorPhoneNumber.substring(authorPhoneNumber.length - 6)` gives the same result.

File: ts/components/conversation/ContactName.tsx (hunk 1 above)
Comment (BeaudanBrown) [reply ts/components/conversation/ContactName.tsx]: Chucked something else up but not sure how I feel about it

File: ts/components/conversation/ContactName.tsx [hunk 3]
```diff
@@ -15,22 +15,30 @@ interface Props {
...
-      ) : (
-        <span className={`${prefix}__profile-name`}>
+    const styles = (boldProfileName
+      ? {
+          fontWeight: 'bold',
+        }
+      : {}) as React.CSSProperties;
+    const profileElement = shouldShowProfile ? (
+      <span style={styles} className={`${prefix}__profile-name`}>
+        <b>
```
Comment (sachaaaaa) [inline ts/components/conversation/ContactName.tsx]: You can drop the `<b>` now :)

**Pull request merged**